application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/tree/TraceNode.java

```diff
@@ -37,10 +37,12 @@
 import java.util.List;
 
 public class TraceNode {
+	public static final TraceNode EMPTY = new TraceNode(null, 0, null);
 	private final int value;
 	private final String name;
 	private final String packageName;
 	private final List<TraceNode> children = new ArrayList<>();
+	private boolean valid = true;
```

Comment:
This is a weird abstraction. Is this really necessary?

Reply:
I've done a cleanup